contracts/interfaces/IQuestChain.sol

```diff
@@ -9,25 +9,25 @@ interface IQuestChain {
     event QuestChainEdited(address indexed editor, string details);
     event QuestCreated(
```

Comment:
Should we rename the events to be "QuestsCreated" or "MultipleQuestsCreated" for clarity ?

Reply:
Hmm. We may then need to update the method names also.

Reply:
Yeah, I couldn't make up my mind on the rename. Whatever you suggest. 
Alternate to `MultipleQuestsCreated ` can be just `QuestsCreated`.

Reply:
@parv3213 sure lets go with `QuestsCreated` and also method names can be plural as well.
Would make it easier to differentiate in terms of old vs new abi also.

Reply:
aye aye